### DISCUSSION TITLE
1.51 crashing LHR cards, drivers

### DISCUSSION THREAD

**Discussion by p777997** [Lolliedieb/lolMiner-releases#1595] [General]
on 1.51 nearly 20% of LHR cards in my inventory are randomly crashing with "GPU crashed" and "memory error".

after a crash mining can be still done with some 25-30% hashrate, but driver seems to be corrupted as showing 0% fans and no load - only reboot helps.

no settings help to stabilize - switching to single ETH, even with minimum memory clock, core clock, etc.

have absolutely no issues with crashing on 1.49 even with high memory clocks and dual mining.

**Comment by KatilTako77** [comment]
i have the exactly same problem here. Did you solve it?

**Reply by p777997** [reply]
nope. 1.51a still has same issue. among hundreds of LHR cards 25-30% get crashed in a day or two regardless of the settings used even on minimum. these that are unstable - as a temporary solution i'm switching back to 1.49 that has no issues with crashing.

**Reply by yekadzun** [reply]
Hello! Tested 1.51a dual TON-ETH and I see more crash LHR card (3060 colorful), its unstable version
